src/engine/ruleset/decoders/f5/f5-apm.yml

```diff
@@ -0,0 +1,375 @@
+  logpar:
+    # crond
+    # March 2016/03/12 03:17:42 seq high crond[5738]: (ccaecat) veleumi
+    # July 2016/07/18 18:40:50 lum high CROND[1675]: (sitvolup) CMD (cancel)
+    # January 2019/01/19 13:25:23 volup very-high crond[4071]: (iconsequ) CMD (block)
+    # April 2019/04/29 14:43:23 Nequepo high CROND[2977]: (emac) CMD (cancel)
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <event.code>[<process.pid>]:<~/ignore/ >\(<~tmp.user_name>\) (?CMD \(<~tmp.user_command>\))<?~tmp.rsa.db.index>
+
+    # iRules : https://clouddocs.f5.com/api/irules/
+    # October 2019/10/18 03:14:14 tem very-high giatnula[71]: Rule: enimadmi <<qui>: APM_EVENT=deny | aecon | sedq ***failure***
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <event.code>[<process.pid>]:<~/ignore/ ><~tmp.rsa.internal.messageid>:<~/ignore/ ><~tmp.rsa.misc.rule_name> \<\<<~tmp.rsa.internal.event_desc>>:<~/ignore/ >APM_EVENT=<event.action> | <user.name> | <~> ***<~tmp.rsa.misc.result>***
+
+    # syslog-ng - auditd - SMTP
+    # March 2017/03/18 18:24:33 ptasnula high syslog-ng[2638]: ill
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <event.code>[<process.pid>]:<~/ignore/ ><~tmp.rsa_misc_client>
+    # November 2019/11/01 10:16:48 erc low tasnu: [syslog-ng]
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <~tmp.rsa_misc_client>:<~/ignore/:> [<event.code>]
+
+    # April 2017/04/02 01:27:07 caboNem medium laudan[7589]: 01490107: :oconse: mag: AD module: authentication with 'tob' failed: Client 'dolores2519.mail.host' not found in Kerberos database, principal name:deF itempo
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <~>[<process.pid>]<~/ignore/:> <event.code>:<~/ignore/ ><~content>
+    # May 2016/05/08 07:27:59 labor low isqu: 01490167: :uis: Current snapshot ID: idolore updated inside session db for access profile: onse
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <~>:<~/ignore/:> <event.code>:<~/ignore/:><~content>
+
+normalize:
+
+  # 011f0005
+  # March 2017/03/04 11:21:59 unte very-high ueipsa[748]: 011f0005: :cti: failure (Client side: vip=https://www5.example.com/olli/rever.html?rsp=oluptat#metco profile=ipv6-icmp pool=edolorin client_ip=10.104.110.134)
+  - check: event.code==011f0005
+    logpar:
+      - ~content: <~/ignore/:><~>:<~/ignore/ ><~tmp.rsa_misc_result> <~/literal/(Client side:><~/ignore/ >vip=<~url/uri> profile=<network.protocol> pool=<~> client_ip=<source.ip>\)
+    map:
+      - url: +ef_rename/$~url
+      - rsa.misc.result: $~tmp.rsa_misc_result
+
+  # 01490106
+  # oremagn[2121]: 01490106: :itseddo: uptatev: AD module: authentication with 'oditem' failed in allow: Preauthentication failed, principal name: inimaven. failure olor
+  - check: event.code==01490106
+    logpar:
+      - ~content: $DEF_SESSION:<~/ignore/:> <~>:<~/ignore/:> <~>'<~tmp.user>' <~>:<~/ignore/:> <~>, <~>. <~tmp.rsa_misc_result> <~>
+    map:
```

Comment:
The way the examples are displayed might be confusing. Which part of the log is the `~content`?

Reply:
Same observation for all the cases.